MeasureAuthoringTool/mat-drupal
# Feature/drupal news and alerts

## Description
Created News & Alerts view. Configured infobox display mode for homepage and teaser list for News & Alerts page. Configured pathauto for article content type, and added menu_trail_by_path module so that article items are nested under News & Alerts without needing to be added to the menu.

## To Test
1. Update composer dependencies `composer install`
2. Import Drupal config changes `docker-compose exec drupal drush cim`
3. Compile PL `cd html/themes/custom/emat && npm start`
4. Clear Drupal caches `docker-compose exec drupal drush cr`
5. Create at least 5 test articles `/node/add/article`
6. Visit homepage, see that the "Latest News & Alerts" block is being displayed as expected
7. Create a Basic Page called "News & Alerts", make sure it's path is `/news-alerts`
8. Visit that page, see that the News & Alerts teaser list is being displayed as exected, and that the Show More button loads additional articles via ajax.
